disnake/widget.py

```diff
@@ -369,11 +379,11 @@ def json_url(self) -> str:
         return f"https://discord.com/api/guilds/{self.id}/widget.json"
```

Comment:
honestly we should probably version this api while we're here.

Reply:
I also considered that, but the client deliberately uses the unversioned url so it's be better to replicate that here, imo.

Reply:
it wouldn't be so bad if the default version wasn't the oldest version too.